securesystemslib/gpg/common.py

```diff
@@ -632,7 +632,10 @@ def parse_signature_packet(data, supported_signature_types=None,
     raise ValueError("Signature version '{}' not supported, must be one of "
         "{}.".format(version_number, SUPPORTED_SIGNATURE_PACKET_VERSIONS))
 
-  # here, we want to make sure the signature type is indeed PKCSV1.5 with RSA
+  # Here, we want to make sure the signature type is indeed a "signature of a
+  # binary document", type 0x00 from RFC 4880 section 5.2.1.  This is as
+  # opposed to a revocation signature, or a signature of one key over another,
+  # or a persona certification, etc.
```

Comment:
`SIGNATURE_TYPE_BINARY` is only the default signature type that's checked for in `parse_signature_packet`. But you can pass other `supported_signature_types` as we e.g. do in `_assign_certified_key_info` (`SIGNATURE_TYPE_CERTIFICATES`) and `_get_verified_subkeys` (`SIGNATURE_TYPE_SUB_KEY_BINDING`).